saos-restful-web-api/src/test/java/pl/edu/icm/saos/api/dump/enrichmenttag/DumpEnrichmentTagControllerTest.java

```diff
@@ -0,0 +1,199 @@
+package pl.edu.icm.saos.api.dump.enrichmenttag;
+
+import static org.hamcrest.Matchers.hasSize;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static org.springframework.test.web.servlet.setup.MockMvcBuilders.standaloneSetup;
+import static pl.edu.icm.saos.api.ApiConstants.PAGE_NUMBER;
+import static pl.edu.icm.saos.api.ApiConstants.PAGE_SIZE;
+import static pl.edu.icm.saos.common.json.JsonNormalizer.normalizeJson;
+
+import java.util.List;
+
+import org.assertj.core.util.Lists;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.ResultActions;
+
+import pl.edu.icm.saos.api.ApiTestConfiguration;
+import pl.edu.icm.saos.api.search.parameters.ParametersExtractor;
+import pl.edu.icm.saos.api.services.interceptor.RestrictParamsHandlerInterceptor;
+import pl.edu.icm.saos.common.testcommon.category.SlowTest;
+import pl.edu.icm.saos.persistence.PersistenceTestSupport;
+import pl.edu.icm.saos.persistence.common.TestObjectContext;
+import pl.edu.icm.saos.persistence.common.TestPersistenceObjectFactory;
+import pl.edu.icm.saos.persistence.enrichment.EnrichmentTagRepository;
+import pl.edu.icm.saos.persistence.enrichment.model.EnrichmentTag;
+import pl.edu.icm.saos.persistence.search.DatabaseSearchService;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration(classes =  {ApiTestConfiguration.class})
+@Category(SlowTest.class)
+public class DumpEnrichmentTagControllerTest extends PersistenceTestSupport {
+
+    @Autowired
+    private ParametersExtractor parametersExtractor;
+
+    @Autowired
+    private DatabaseSearchService databaseSearchService;
+
+    @Autowired
+    private DumpEnrichmentTagsListSuccessRepresentationBuilder dumpEnrichmentTagsListSuccessRepresentationBuilder;
+
+    @Autowired
+    private TestPersistenceObjectFactory testPersistenceObjectFactory;
+    
+    @Autowired
+    private EnrichmentTagRepository enrichmentTagRepository;
+
+    private TestObjectContext testObjectContext;
+
+    private MockMvc mockMvc;
+    
+    
+    private List<EnrichmentTag> enrichmentTags = Lists.newArrayList();
+    
+    private final static String DUMP_ENRICHMENT_PATH = "/api/dump/enrichment";
```

Comment:
Is it correct path?
You have in the controller: `@RequestMapping("/api/dump/enrichments")`